src/app/find-help/material-support-search/material-support-search-form/material-support-search-form.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Output, EventEmitter } from '@angular/core';
+import { MaterialAidOffer } from '../../../../api/model/materialAidOffer';
+
+const categories = Object.entries(MaterialAidOffer.CategoryEnum).map(([key, value]) => ({
+  code: key,
+  label: value,
+}));
+
+export interface MaterialSupportQuery {
```

Comment:
Delete this type, use `MaterialAidOfferSearchCriteria` from the `api` folder directly.